tests/phpunit/tests/rest-api/test-notification-controller.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Unit tests covering WP_REST_Notification_Controller functionality.
+ */
+class WP_Test_REST_Notification_Controller extends WP_Test_REST_Post_Type_Controller_Testcase {
```

Comment:
@Sephsekla should I use namespaces in the tests or name test classes with a prefix?